server_v2/lib/resumes/main.py

```diff
@@ -0,0 +1,54 @@
+from lib.OpenAI_Lib.main import *
+from helpers import *
+
+class Analyser:
+    def __init__(self, **kwargs):
+        self.model = kwargs.get('model_name')
+        pass
+
+    def evaluate_characterstics(jinja_templated_resume_text=""):
+
+        # Get user input for the template fields
+        position_title = input("Enter Position Title: ")
+        duration = input("Enter Duration in Years: ")
+        industry = input("Enter Industry: ")
+        primary_skills = input("Enter Primary Skills (comma-separated): ")
+        secondary_skills = input("Enter Secondary Skills (comma-separated): ")
+        degree = input("Enter Degree: ")
+        field_of_study = input("Enter Field of Study: ")
+        project_keywords = input("Enter Project Keywords (comma-separated): ")
+        misc_keywords = input("Enter Miscellaneous Keywords (comma-separated): ")
+        location_preference = input("Enter Preferred Location: ")
+        language_proficiencies = input("Enter Language Proficiencies (comma-separated): ")
+
+        # Render the template with user inputs
+        filled_template = jinja_templated_resume_text.render(
+            position_title=position_title,
+            duration=duration,
+            industry=industry,
+            primary_skills=primary_skills,
+            secondary_skills=secondary_skills,
+            degree=degree,
+            field_of_study=field_of_study,
+            project_keywords=project_keywords,
+            misc_keywords=misc_keywords,
+            location_preference=location_preference,
+            language_proficiencies=language_proficiencies
+        )
+
+        prompt = filled_template + "\n\n" + """Action- 
+                Generate a prompt based on the provided details to query from the provided resume, whether the profile is suitable or not. Return only JSON output. Json output should contain the following fields-
```

Comment:
Create new dir and add  templates/resumes/outputs/software_dev.txt

This part of string should be loaded from the above file